src/product.hpp

```diff
@@ -7,7 +7,7 @@
 /**
  * USB Power Detection Pin TODO
  */
-#define SF_USB_PWR_DETECT_PIN   45
+#define SF_USB_PWR_DETECT_PIN   51
```

Comment:
This still will not work - have you tried `digitalRead(51)` to see if it will behave correctly?

Reply:
I'll try that tomorrow, was basing this off of the documentation today since I wasnt in the lab.

Reply:
I will remind you that I am less interested in this getting done quickly and more interested in it getting done correctly.  Please take the time to test all of the code before you submit these changes for review.